homepage/homepage/app/(docs)/docs/[framework]/[[...slug]]/project-setup/providers/react-native-expo.mdx

```diff
@@ -0,0 +1,143 @@
+export const metadata = { title: "React Native Expo Providers" };
+
+import { CodeGroup } from "@/components/forMdx";
+
+# React Native Expo Providers
+
+## Introduction
+
+`<JazzProvider />` is the core component that connects your Expo application to Jazz. It handles:
+
+- **Data Synchronization**: Manages connections to peers and the Jazz cloud
+- **Local Storage**: Persists data locally between app sessions
+- **Schema Management**: Associates your data models with Jazz
```

Comment:
The provider only provides the APIs for the AccountSchema, not the entire data model.

The data model is managed via `useCoState`